client/src/driver/BitBakeProjectScanner.ts

```diff
@@ -43,7 +43,6 @@ export class BitBakeProjectScanner {
   onChange: EventEmitter = new EventEmitter()
 
   private readonly _bitbakeScanResult: BitbakeScanResult = { _classes: [], _includes: [], _layers: [], _overrides: [], _recipes: [], _workspaces: [], _confFiles: [] }
-  private _shouldDeepExamine: boolean = false
```

Comment:
After more experiments, I think we can still keep this setting because the command `bitbake-layers show-recipes -f <recipe>` gives the most accurate path for a known recipe name. The downside is it significantly impacts the performance since it runs the command for each recipe that didn't get a matching path. I will run my logic for inferring the path when the `deep examine` is not activated to give close results. Also, I will add the performance warning in the setting description.

Reply:
Now it does the infer when the `deep examine` is off.